src/docs/resources/libraries/README.md

```diff
@@ -17,7 +17,7 @@ The below table lists the different .NET libraries used in Orchard Core:
 | [Fluid.Core](https://github.com/sebastienros/fluid) | .NET Liquid template engine. | 2.5.0 | [MIT](https://github.com/sebastienros/fluid/blob/dev/LICENSE) |
 | [GraphQL](https://github.com/graphql/graphiql) | GraphiQL & GraphQL. | 4.6.1 | [MIT](https://github.com/graphql/graphiql/blob/main/LICENSE) |
 | [HtmlSanitizer](https://github.com/mganss/HtmlSanitizer) | Cleans HTML to avoid XSS attacks. | 8.1.812-beta | [MIT](https://github.com/mganss/HtmlSanitizer/blob/master/LICENSE.md) |
-| [Image Sharp](https://github.com/SixLabors/ImageSharp.Web) | Middleware for ASP.NET-Core for image manipulation. | 3.1.0 |[Apache-2.0](https://github.com/SixLabors/ImageSharp.Web/blob/master/LICENSE) |
+| [ImageSharp](https://github.com/SixLabors/ImageSharp.Web) | Middleware for ASP.NET-Core for image manipulation. | 3.1.2 |[Apache-2.0](https://github.com/SixLabors/ImageSharp.Web/blob/master/LICENSE) |
```

Comment:
I suggest merging this PR #15822 too

Reply:
That's not applicable since those packages are only used in `main`.

Reply:
While ImageSharp version is `3.1.2` I think it's updated recently, right?

Reply:
Sorry, I don't understand. Note that this is a patch release on top of 1.8.2. Anything done in `main` since that is intentionally not included.

Reply:
> Anything done in the main since that is intentionally not included.

I see, but how was the ImageShape version updated above, while it was merged last week #15767

Reply:
That PR was merged to `main`. This PR is not started from `main`, but `release/1.8.3`. We don't want to release `main`, since that contains a lot of stuff for 1.9 and it's not ready yet. We do want to release the two dependency updates in this PR ASAP and not wait for anything else, since they're security vulnerabilities. So, the update to the version of these needed to be done a second time here too.